agent/acs/handler/payload_handler.go

```diff
@@ -274,7 +286,7 @@ func (payloadHandler *payloadRequestHandler) handleUnrecognizedTask(task *ecsacs
 	}
 
 	// Only need to stop the task; it brings down the containers too.
-	eventhandler.AddTaskEvent(api.TaskStateChange{
+	payloadHandler.taskHandler.AddTaskEvent(api.TaskStateChange{
```

Comment:
yes this is currently the only use, although i'm not that reimplementing the all the event submission logic would be a good idea. @aaithal any comments?